### ISSUE TITLE
pipenv run seems to capture or redirect my stdin/stdout

### ISSUE THREAD

**Issue by erinxocon** [kennethreitz/pipenv#365]
I've been using pipenv for a new personal project and it's been working great!  However when I run my script using `pipenv run python server.py` I don't see any of my logging debug statements.  If I activate the virtualenv manually and then run `python server.py` it does output all of my logging statements to the stdout as I'd expect.  Is this expected behavior?  Any help would be greatly appreciated.  Thanks!

**Comment by nateprewitt** [comment]
Hey @erinxocon, are you using a `Logger` to write your logging statements? I know @kennethreitz set up a few changes for certain things being written to stderr and stdout but I'm not sure if that would affect your use case.

Right now, pipenv seems to supply what I'd expect from something like `pipenv run python -c "print('hello')"`. If you could provide a more specific repro, that would be incredibly helpful. Thanks!

**Comment by erinxocon** [comment]
@nateprewitt I am indeed using the standard python logger, the code currently isn't public due to using some proprietary code from work :/  I'll try and upload a simple example of the issue later today!

**Comment by erinxocon** [comment]
@nateprewitt print statements still seem seem to work as expected

**Comment by nateprewitt** [comment]
Thanks for confirming that @erinxocon :) So I threw together another quick test with the standard logging module and can't reproduce the problem there either.

```python
import logging

logging.basicConfig(level=logging.DEBUG)

logging.warning('warn')
logging.debug('error')
logging.info('info')
```

Which returns this output from `pipenv run python test.py`:

```
WARNING:root:warn
DEBUG:root:error
INFO:root:info
```

If that functions the same for you, we will need that simpler example. It definitely doesn't need to be the proprietary code, just a similar example of how you're using the logger.

**Comment by erinxocon** [comment]
@nateprewitt I just ran a similar test.  
`pipenv run python -c "import logging; logging.basicConfig(level=logging.DEBUG); logging.warning('warn'); logging.debug('debug')"`

Which output: 
`WARNING:root:warn
DEBUG:root:debug`

So it must be something to do with how I have the logger configured :/  I'll troubleshoot and if I'm still having problems I'll upload an example.

**Comment by erinxocon** [comment]
@nateprewitt I figured it out!  In talking with @kennethreitz recently I switched to fish shell.  I was previously using Ken's autoenv tool to load some environment variables as well as activate the virtual environment for me upon entering the directory.  The .env file I had was formatted for zsh/sh script so my environment variables weren't being exported when I entered the directory due to the different syntax.  One of those variables was Log level, which defaulted to WARNING in my code if it couldn't find the env variable, hence not showing any of my debug statements.  

Thanks for your help appreciate the response!
